docs/versioned_docs/version-7.0.x/configuration/overview.md

```diff
@@ -7,7 +7,8 @@ title: Overview
 
 ### Generating a Cookie Secret
 
-To generate a strong cookie secret use `python -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(16)).decode())'`
+To generate a strong cookie secret use `python -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(16)).decode())'`  
+You can use this [Online Python Compiler](https://www.tutorialspoint.com/execute_python_online.php) if you don't have python available on your own system.
```

Comment:
I don't want to recommend people generate their secrets on internet services if they don't have a local python. Whatever that service is will now know people's cookie secret, I can't vouch for their trustworthiness. 

I'd truthfully prefer they just mash the keyboard randomly 😄